### PR TITLE
NF: Adds fetcher for cerebellar peduncle waypoint ROIs.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

**Comment by 36000** [comment]
It looks like this only adds the probability maps?

**Comment by arokem** [comment]
I didn't change the file names that I received from Sivan, but they are in fact waypoint ROIs. We don't have probability maps for the CPs.

**Comment by 36000** [comment]
Here is what they look like on the stanford hardi subject
[cerebellar_test.zip](https://github.com/yeatmanlab/pyAFQ/files/11602932/cerebellar_test.zip)

**Comment by arokem** [comment]
Thanks for trying it out! I am not sure that subject is the best example to try, given cerebellum was slightly cut off in that acquisiton: 

Screenshot 2023-05-30 at 12 23 24 PM

Could you share the script you ran to get these? Maybe we can try this with an HBN subject that has the whole cerebellum. I will say, it looks like the bundles merge into some other bundles on their superior side as well, so we might want to consider some kind of exclusion criterion, or cut them at the ROIs per default.

**Comment by 36000** [comment]
I pushed the code to generate those html files, so all one needs to do is add CEREBELLAR_BUNDLES into bundle_info in the AFQ object, like:
```
bundle_info = CEREBELLAR_BUNDLES + BUNDLES + CALLOSUM_BUNDLES
```

**Comment by arokem** [comment]
The data Yossinger et al. worked with is apparently all available here: https://osf.io/qjxkb/?view_only=0b8b9017955a41d69168ad9770001538. Sivan told me that subject 25 was a particularly good example.

**Comment by arokem** [comment]
After some more experiments, and digging through Sivan's matlab code, @tmgomez and I think that the way to do this is something like this (note crossing of SCP and MCP, and addition of exclusion ROIs):

```

cp_rois = afd.read_cp_templates()
cp_bundles = abd.BundleDict(
            {
                "ICP_L": {
                    "include": [
                        cp_rois["ICP_L_inferior_prob"],
                        cp_rois["ICP_L_superior_prob"],
                    ],
                    "cross_midline": False,
                },
                "ICP_R": {
                    "include": [
                        cp_rois["ICP_R_inferior_prob"],
                        cp_rois["ICP_R_superior_prob"],
                    ],
                    "cross_midline": False,
                },
                "MCP_L": {
                    "include": [
                        cp_rois["MCP_L_inferior_prob"],
                        cp_rois["MCP_R_superior_prob"],
                    ],
                    "exclude": [
                        cp_rois["SCP_L_inter_prob"],
                    ],
                    "cross_midline": True,
                },
                "MCP_R": {
                    "include": [
                        cp_rois["MCP_R_inferior_prob"],
                        cp_rois["MCP_L_superior_prob"],
                    ],
                    "exclude": [
                        cp_rois["SCP_R_inter_prob"],
                    ],
                    "cross_midline": True,
                },
                "SCP_L": {
                    "include": [
                        cp_rois["SCP_L_inferior_prob"],
                        cp_rois["SCP_L_inter_prob"],
                        cp_rois["SCP_R_superior_prob"],
                    ],
                    "exclude": [
                        cp_rois["SCP_L_superior_prob"],
                    ],
                    "cross_midline": True,
                },
                "SCP_R": {
                    "include": [
                        cp_rois["SCP_R_inferior_prob"],
                        cp_rois["SCP_R_inter_prob"],
                        cp_rois["SCP_L_superior_prob"],
                    ],
                    "exclude": [
                        cp_rois["SCP_R_superior_prob"],
                    ],
                    "cross_midline": True,
                },
            }
        )

cp_afq = GroupAFQ(
            bids_path=bids_path,
            preproc_pipeline="qsiprep",
            brain_mask_definition=brain_mask_definition,
            # mapping_definition=mapping_definition,
            tracking_params={
                "n_seeds": 4,
                "directions": "prob",
                "odf_model": "CSD",
                "seed_mask": RoiImage(),
            },
            bundle_info=cp_bundles,
        )
```

File: examples/howto_examples/cerebellar_peduncles.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+peduncles. There are three CPs: The ICP, the MCP, and the SCP. Each CP is
+defined by two inclusion ROIs and one exclusion ROI. The Inferior CPs are
+defined by inclusion ROIs. They do not decussate, so "cross_midline" is set to
+False. The Superior CPs are defined by two inclusion ROIs and an exclusion ROI,
+where each SCP's most superior inclusion ROI is the other SCP's exclusion ROI.
+They decussate, so "cross_midline" is set to True. The Middle CPs are defined
+by two inclusion ROIs and they use the SCP intermediate ROIs as exclusion ROIs.
+"""
+
+cp_bundles = abd.BundleDict(
```
Comment (36000) [inline examples/howto_examples/cerebellar_peduncles.py:48]: add this bundle dict as a constant in api.bundle_dict file (CerebellarBundleDict)

**Review by 36000 — APPROVED** [review]

File: examples/howto_examples/cerebellar_peduncles.py (hunk 1 above)
Comment (arokem) [reply examples/howto_examples/cerebellar_peduncles.py:48]: Do you mean as a derived class from BundleDict? Along the lines of the PediatricBundleDict

Comment (36000) [reply examples/howto_examples/cerebellar_peduncles.py:48]: Hmmm.... Maybe a better idea is to have a function which returns this.
```
def get_cerebellar_bundles():
    cp_rois = afd.read_cp_templates()
    cp_bundles = abd.BundleDict(
    {
        "ICP_L": {
            "include": [
                cp_rois["ICP_L_inferior_prob"],
                cp_rois["ICP_L_superior_prob"],
            ],
            "cross_midline": False,
        },
        "ICP_R": {
            "include": [
                cp_rois["ICP_R_inferior_prob"],
                cp_rois["ICP_R_superior_prob"],
            ],
            "cross_midline": False,
        },
        "MCP_L": {
            "include": [
                cp_rois["MCP_L_inferior_prob"],
                cp_rois["MCP_R_superior_prob"],
            ],
            "exclude": [
                cp_rois["SCP_L_inter_prob"],
            ],
            "cross_midline": True,
        },
        "MCP_R": {
            "include": [
                cp_rois["MCP_R_inferior_prob"],
                cp_rois["MCP_L_superior_prob"],
            ],
            "exclude": [
                cp_rois["SCP_R_inter_prob"],
            ],
            "cross_midline": True,
        },
        "SCP_L": {
            "include": [
                cp_rois["SCP_L_inferior_prob"],
                cp_rois["SCP_L_inter_prob"],
                cp_rois["SCP_R_superior_prob"],
            ],
            "exclude": [
                cp_rois["SCP_L_superior_prob"],
            ],
            "cross_midline": True,
        },
        "SCP_R": {
            "include": [
                cp_rois["SCP_R_inferior_prob"],
                cp_rois["SCP_R_inter_prob"],
                cp_rois["SCP_L_superior_prob"],
            ],
            "exclude": [
                cp_rois["SCP_R_superior_prob"],
            ],
            "cross_midline": True,
        },
    }
    return cp_bundles
)
```

**Comment by 36000** [comment]
I just rebased this and will merge it once the tests pass

**Pull request merged**